docs/index.md

```diff
@@ -8,7 +8,8 @@ description: |-
 
 # Juju Provider
 
-The provider can be used to interact with [Juju][0] - an open source orchestration engine by Canonical.
+The provider can be used to interact with [Juju][0] - an open source orchestration engine by Canonical.  
+Additionally, the provider supports interactions with [JAAS][1] - an orchestrator of Juju controllers.
```

Comment:
Perhaps this is a good time to update the description in this section: https://github.com/juju/terraform-provider-juju/blob/5f7b9c08783490a6d2cd7b231248ee869cdc1c27/docs/index.md#client-credentials

To include they are used to connect to JAAS? It's not clear in the Authentication section which values will work for JAAS.

Reply:
I've added a note, let me know if I should expand it.